app/helpers/email_helper.rb

```diff
@@ -1,5 +1,9 @@
 module EmailHelper
   def current_subdomain
-    Apartment::Tenant.current == "public" ? "" : Apartment::Tenant.current 
+    if Hours.single_tenant_mode?
+      ""
+    else
+      Apartment::Tenant.current == "public" ? "" : Apartment::Tenant.current
+    end
```

Comment:
The code that Jurre supplied does the same as your code in that diff. So in two lines it will work :)